app/templates/main/faq.html

```diff
@@ -3,103 +3,142 @@
 {% block content %}
 <div class="container-fluid">
 
-    <h2 class="text-center">Frequently Asked Questions</h2>
+    <h2 class="text-center" id="faq-header">Frequently Asked Questions</h2>
+    <h4><a href="#faq-one">What does FOIL mean?</a></h4>
+    <h4><a href="#faq-two">What kinds of records do I have the right to see?</a></h4>
+    <h4><a href="#faq-three">Are there any public records that I can’t see?</a></h4>
+    <h4><a href="#faq-four">Do I have to explain why I want the records?</a></h4>
+    <h4><a href="#faq-five">How much does it cost to get access to government records?</a></h4>
+    <h4><a href="#faq-six">How long does it take to get access to government records?</a></h4>
+    <h4><a href="#faq-seven">Can I make an anonymous request?</a></h4>
+    <h4><a href="#faq-eight">How do I know which agency handles the records I need?</a></h4>
+    <h4><a href="#faq-nine">How do I make a request?</a></h4>
+    <h4><a href="#faq-ten">I asked for a copy of public records and the agency refused my request, saying that
+        it was too broad. What can I do?</a></h4>
+    <h4><a href="#faq-eleven">How do I track a request?</a></h4>
+    <h4><a href="#faq-twelve">What if I have multiple requests that I want to track or would like to see what other
+        requests have been submitted?</a></h4>
+    <h4><a href="#faq-thirteen">What if I don’t agree with an agency’s response?</a></h4>
     <br>
 
-    <h4>What does FOIL mean?</h4>
+    <h4 id="faq-one">What does FOIL mean?</h4>
 
     <p>FOIL stands for the Freedom of Information Law. The New York State Freedom of Information Law
         requires government agencies to provide records to the public upon request. Government records
         affect the lives of each New Yorker, and providing easy access to the records makes government work
-        better.</p>
+        better.
+        <br />
+        <a href="#faq-header">^ Back to top</a></p>
```

Comment:
In place of this, maybe use: `<span class="glyphicon glyphicon-chevron-up></span>`